modules/ROOT/pages/migration-from-7x.adoc

```diff
@@ -3,9 +3,665 @@
+This change improves security and aligns with DOMPurify's recommended defaults. However, existing content and integrations that relied on the previous, less strict sanitization behavior may be impacted.
+====
+
+**Key Changes**:
+
+* **DOMPurify upgraded to 3.2.6**
+* **`SAFE_FOR_XML` enabled** — This setting enforces stricter handling of comments and attribute values, preventing certain XSS vectors.
+* **Content Impact** — HTML comments containing tags, Internet Explorer conditional comments, and attributes with HTML-like values may now be removed during sanitization. Content that was previously allowed may be stripped.
+
+**Impact**: This change improves security by preventing potential XSS attacks through comments and attributes that were previously allowed. However, it may also result in content being stripped or altered unexpectedly.
+
+**Migration Steps:**
+
+* Review workflows and test content that previously relied on relaxed sanitization.
+* {productname} now provides the xref:content-filtering.adoc#allow_html_in_comments[Content Filtering: allow_html_in_comments option] option. Enabling this option allows HTML tags in comments with sanitization still enabled. 
+
+[WARNING]
+Using `allow_html_in_comments` increases the risk of XSS vulnerabilities. xref:security.adoc#allow_html_in_comments[allow_html_in_comments] is not recommended for production use unless you fully understand the implications and have appropriate security measures in place.
+
+.Example: Content Differences
+|===
+|Content |{productname} 7 Output |{productname} {release-version} Output
+
+|`<div><!-- <p>Hello World</p> --></div>`
+|`<div><!-- <p>Hello World</p> --></div>`
+|`<div></div>`
+
+|`<iframe><!-- Hello World --></iframe>`
+|`<p><iframe sandbox=""><!-- Hello World --></iframe></p>`
+|``
+
+|`<script><!-- Hello World --></script>`
+|`<script><!-- Hello World --></script>`
+|``
+|===
+
+[NOTE]
+====
+For information on disabling DOMPurify sanitization (not recommended), see xref:security.adoc#xss_sanitization-option[xss_sanitization option].
+====
+
+== Core Changes
+
+[[split-button-css-breaking-change]]
+=== Split button CSS structure updates
+// #TINY-8665
+
+{productname} {productmajorversion} updates the internal structure of split buttons. Split buttons now render as two separate components for the main action and the dropdown chevron.
+
+**Impact**: This change only affects users with custom skins.
+
+**Migration steps**:
+
+If you have a custom skin, rebuild it using the {productname} {productmajorversion} codebase. See xref:creating-a-skin.adoc[Creating a Skin] for guidance on the custom skin development process.
+
+**Migration checklist:**
+
+* [ ] Identify if you are using a custom skin.
+* [ ] Rebuild your custom skin using the {productname} {productmajorversion} codebase.
+* [ ] Test your custom skin with split buttons in {productname} {productmajorversion}.
+* [ ] Refer to xref:creating-a-skin.adoc[Creating a Skin] for details.
+
+**Code examples**
```

Comment:
I think the examples may not be relevant to the migration steps for split buttons (adding a `chevronTooltip` from what I can see). Should we add a heading here to indicate that this should be the new structure for split buttons? Or should we move this to the split button page?

Reply:
removed as discussed.